openstates/de/bills.py

```diff
@@ -1,693 +1,329 @@
-                raise Exception("bill title is empty")
-            bill['title'] = bill['summary']
-            # This added to help hack around the page that's missing
-            # the bill title.
-
-        self.save_bill(bill)
-
-    def scrape_vote(self, url,
-                    re_digit=re.compile(r'\d{1,3}'),
-                    re_totals=re.compile(
-                        r'(?:Yes|No|Not Voting|Absent):\s{,3}(\d{,3})', re.I)):
-        namespaces = {"re": "http://exslt.org/regular-expressions"}
-        try:
-            html = self.urlopen(url)
-            doc = lxml.html.fromstring(html)
-        except scrapelib.HTTPError as e:
-            known_fail_links = [
-                "http://legis.delaware.gov/LIS/lis146.nsf/7712cf7cc0e9227a852568470077336f/cdfd8149e79c2bb385257a24006e9f7a?OpenDocument",
-                'http://legis.delaware.gov/LIS/lis147.nsf/7712cf7cc0e9227a852568470077336f/5f86852ea6649fa285257d08001bbe06?OpenDocument'
-            ]
-            if "404" in str(e.response):
-                # XXX: Ugh, ok, so there's no way (that I could find quickly)
-                #      to get the _actual_ response (just "ok") from the object.
-                #      As a result, this. Forgive me.
-                #            -PRT
-                # XXX: THERE SHALL BE NO FORGIVENESS FOR PAULTAG!!!!
-                #
-                #       Just kidding. I blame Delaware.
-                #            -TWN
-                if url in known_fail_links:
-                    msg = 'Recieved a bogus 22/404 return code. Skipping vote.'
-                    self.warning(msg)
-                    return
-            raise
-
-        if 'Committee Report' in lxml.html.tostring(doc):
-            # This was a committee vote with weird formatting.
-            self.info('Skipping committee report.')
-            return
 
-        xpath = ("//font[re:match(., '^(Yes|No|Not Voting|Absent):', 'i')]"
-                 "/ancestor::tr[1]")
 
-        # Get the vote tallies.
+        base_url = "http://legis.delaware.gov"
+        text_base_url = "http://legis.delaware.gov/LIS/lis{session}.nsf/vwLegislation/{bill_id}/$file/legis.html?open"
+        page = self.lxmlize(link)
+        bill_id = page.xpath(".//div[@align='center']")
         try:
-            totals = doc.xpath(xpath, namespaces=namespaces)
-            totals = totals[0].text_content()
-
+            bill_id = bill_id[0].text_content().strip()
         except IndexError:
-            # Here the vote page didn't have have the typical format.
-            # Maybe it's a hand edited page. Log and try to parse
-            # the vitals from plain text.
-            self.warning('Found an unusual votes page at url: "%s"' % url)
-            totals = re_totals.findall(doc.text_content())
-            if len(totals) == 4:
-                self.warning('...was able to parse vote tallies from "%s"' % url)
+            self.logger.warning("Can't find bill number, skipping")
```

Comment:
Seems like we get a slew of these errors when the scraper hits nominations. Could we `self.info()` skip instead of `self.warning()` if the scraper sees a nomination? Maybe if it sees "Nomination Information" as the page header, [like here](http://legis.delaware.gov/LIS/lis148.nsf/ac4d906954d0945685257e0b00557ac0/1c8ede0fde8f351485257dda005ba189?OpenDocument)?